modules/crawler.py

```diff
@@ -153,7 +153,7 @@ async def robots(target):
 	print(G + '[+]' + C + ' Looking for robots.txt' + W, end = '')
 
 	try:
-		r_rqst = requests.get(r_url, headers=user_agent, verify=False, timeout=10)
+		r_rqst = requests.get(r_url, headers=user_agent, verify=True, timeout=10)
```

Comment:
if verify is set to `True` here, websites which have broken SSL will fail even if they have a robots file so this needs to be `False`